src/ipfs.io/docs/install/index.html

```diff
@@ -10,118 +10,91 @@
 <div class="col-lg-6 col-lg-offset-3 col-sm-8 col-sm-offset-2" style="font-family: Oxygen; line-height: 1.7">
 {% markdown %}
 
-# Install IPFS
+# Install Go IPFS
```

Comment:
Maybe we should be asking people to install through `ipfs-update`. that's an annoying "bootstrap" problem.